src/app/components/forms/post/post.styl

```diff
@@ -124,6 +124,9 @@
 			border-top: 1px solid var(--theme-bg-subtle)
 			border-bottom: 1px solid var(--theme-bg-subtle)
 
+			> :not(:first-child)
```

Comment:
What is this?

Reply:
On mobile it turns out the buttons dont have any margins between them so it's easy to mispress. This fixes it.

Reply:
Gotcha.